src/v2/view-builder/components/IdentifierFooter.js

```diff
@@ -4,6 +4,16 @@ import { FORMS as RemediationForms } from '../../ion/RemediationConstants';
 import { getForgotPasswordLink } from '../utils/LinksUtil';
 
 export default BaseFooter.extend({
+  postRender () {
```

Comment:
probably worth to make change around line 35 below plus OKTA-245224 now as separated PR?

```
if (this.options.appState.hasRemediationObject(RemediationForms.SELECT_ENROLL_PROFILE)) {
  const signupLink = {
        'type': 'link',
        'label': loc('signup', 'login'),
        'name': 'enroll',
      };
  if (isFunction of this.options.settings.get('registration.click')) {
    signupLink.click = this.options.settings.get('registration.click');
  } else {
    signupLink.actionPath = RemediationForms.SELECT_ENROLL_PROFILE;
  } 
      signupLink.push(signupLink);
}
```

Reply:
Ah this is supposed to happen instead of moving forward in the flow

Reply:
What do you mean by that jira?

Reply:
https://github.com/okta/okta-signin-widget/pull/1747/commits/7c674d4c702494e5a13fc0974c457f048dff0ec7

Reply:
For that, I feel it's more explicit to check on type instead of on the availability of certain value.
(just an optional change)